sources/Application/AppWindow.cpp

```diff
@@ -518,7 +519,9 @@ bool AppWindow::onEvent(GUIEvent &event) {
   unsigned short v = 1 << event.GetValue();
 
   MixerService *sm = MixerService::GetInstance();
-  sm->Lock();
+  // TODO(democloid): this causes a deadlock, verify original intent
```

Comment:
my concern here is if the mutex was not actually being enabled (by mistake) but things actually work fine then it really shouldnt be here if it causes a deadlock and we should just remove it.